packages/create-sitecore-jss/src/templates/nextjs-personalize/.env

```
@@ -1,11 +1,11 @@
 # Your Sitecore CDP REST API base URL
-NEXT_PUBLIC_CDP_API_URL=https://api.boxever.com
+NEXT_PUBLIC_CDP_API_URL=https://api-engage-eu.sitecorecloud.io
```

Should we actually use the dev endpoint? This is non-region specific and actually makes more sense to start with from a dev-perspective.
```suggestion
NEXT_PUBLIC_CDP_API_URL=https://api-engage-dev.sitecorecloud.io
```

---

@ambrauer Good point, replaced